src/modules/cachix.nix

```diff
@@ -12,13 +12,13 @@ in
 
     pull = lib.mkOption {
       type = lib.types.listOf lib.types.str;
-      description = "What caches to pull from.";
-      default = [ ];
+      description = "Which Cachix caches to pull from.";
+      default = [ "devenv" ];
```

Comment:
That means if someone sets a cache, devenv won't be used anymore, which is surprising.

Reply:
Ah, yeah, I thought they'd get merged for some reason.